css/floating-cart.css

```
@@ -0,0 +1,758 @@
+    /* Better iOS Safari viewport handling */
+    max-height: calc(70vh - env(safe-area-inset-bottom, 0));
+  }
+  
+  .cart-items {
+    /* Better iOS Safari viewport handling */
+    max-height: calc(50vh - env(safe-area-inset-bottom, 0));
+  }
+}
+
+/* Android Chrome specific optimizations */
+@media screen and (max-width: 768px) and (-webkit-min-device-pixel-ratio: 1) {
```

Biome (2.1.2)

[error] Don't use unknown media feature names.

Unexpected unknown media feature name.
You should use media feature names defined in the CSS Specifications.

(lint/correctness/noUnknownMediaFeatureName)